content/docs/querying/api.md

```diff
@@ -71,6 +71,7 @@ URL query parameters:
 
 - `query=<string>`: Prometheus expression query string.
 - `time=<rfc3339 | unix_timestamp>`: Evaluation timestamp. Optional.
+- `timeout=<duration>`: Evaluation timeout. Optional. Defaults to value of `-query.timeout` flag.
```

Comment:
I mean, when does it happen that someone actually wants to set a *lower* than default timeout for a query?